s3-file-provider/Package.swift

```diff
@@ -0,0 +1,34 @@
+// swift-tools-version:5.9
```

Comment:
Likewise, shouldn't we prioritise Swift 6 now?

Reply:
yep